lib/pubid/iso/identifier.rb

```diff
@@ -5,12 +5,13 @@ class Identifier < Pubid::Core::Identifier
                   :tctype, :sctype, :wgtype, :tcnumber, :scnumber, :wgnumber,
                   :urn_stage, :dirtype,
                   # supplement for DIR type identifiers
-                  :supplement
+                  :supplement,
+                  :base
 
     TYPE_NAME = {
-      TR: "Technical Report",
-      TS: "Technical Specification",
-      PAS: "Publicly Available Specification",
+      tr: "Technical Report",
```

Comment:
Excellent change that we're now using lower case symbols! I would like to also add the other information as relating to types, i.e.

```ruby
TYPE_NAME = {
  is: {
    long: "International Standard",
    short: "IS"
    valid_stages_codes: [ "00.*", "10.*", ..., "50.*", "60.*" ] # all stages available
  },
  tr: {
    long: "Technical Report",
    short: "TR"
    valid_stages_codes: [ "00", "10", ..., "30.*", "50.*", "60.*" ] # TR skips stage 40.*
  },
  ts: {
    long: "Technical Specification",
    short: "TS"
  },
  pas: ...
}
```

See this for valid stages for different documents:
https://www.iso.org/sites/directives/current/consolidated/index.xhtml
![image](https://user-images.githubusercontent.com/11865/194478303-69dd1861-27d3-4307-8b61-02bb08fb2a1d.jpeg)